converter/internal/common/common.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/pkg/river/rivertypes"
+	"github.com/grafana/agent/pkg/river/token/builder"
+)
+
+// AppendBlock appends Flow component arguments using the convert value
+// override hook to the file we are building.
+func AppendBlock(f *builder.File, name []string, label string, args component.Arguments) {
```

Comment:
This is to save repeated code. It may evolve as more gets added to the convert API or removed if the pattern becomes inconsistent but I'd like it for now.

Reply:
Should `AppendBlock` be called something more descriptive since we already have an append block? Maybe `AppendBlockWithOverride`? Does this cascade?

Reply:
Updated name, good call. What did you mean by cascade in this context?